inference-engine/cmake/features_ie.cmake

```diff
@@ -98,6 +98,8 @@ ie_option (ENABLE_PYTHON "enables ie python bridge build" OFF)
 
 ie_option (ENABLE_V7_SERIALIZE "enables serialization to IR v7" OFF)
 
+ie_option (ENABLE_V10_SERIALIZE "enables experimental serialization to IR v10" OFF)
```

Comment:
Why do you need this? V10 serializer is enabled for default in the master branch.
We don't need to cherry-pick this commit from the release branch.

Reply:
reverted